go-controller/hybrid-overlay/pkg/controller/master_test.go

```diff
@@ -109,15 +105,21 @@ var _ = Describe("Hybrid SDN Master Operations", func() {
 
 			dbSetup := libovsdbtest.TestSetup{}
 			var libovsdbOvnNBClient libovsdbclient.Client
+			var libovsdbOvnSBClient libovsdbclient.Client
+
 			libovsdbOvnNBClient, libovsdbCleanup, err = libovsdbtest.NewNBTestHarness(dbSetup, nil)
```

Comment:
Everywhere you're adding the SB client, you can use:

```
var libovsdbOvnNBClient, libovsdbOvnSBClient libovsdbclient.Client
libovsdbOvnNBClient, libovsdbOvnSBClient, libovsdbCleanup, err = libovsdbtest.NewNBSBTestHarness(dbSetup, nil)
```

Reply:
Yep sorry forgot about that